### PR TITLE
Readme - first version

### PR DESCRIPTION
**Pull request by martapederiva**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/FBT-191

### *What is the problem we're trying to solve*

Creating an MVP readme for our API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,140 @@
...
+    - [Related repositories](#related-repositories)
+  - [Active contributors](#active-contributors)
+  - [License](#license)
+
+## Getting started
+
+### Prerequisites
+
+- [Docker](https://www.docker.com/products/docker-desktop)
+- [.NET Core 3.3](https://dotnet.microsoft.com/download)
```
Comment (pixeltrix) [inline README.md:28]: This should be `.Net Core 3.1`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+
+- [Staging](https://wadss19f8f.execute-api.eu-west-2.amazonaws.com/staging/swagger/index.html) 
+
+- [Production](https://3mhm6sj5o2.execute-api.eu-west-2.amazonaws.com/production/swagger/index.html) 
+
+### Deployment
+
+We have two environments:
+
+- Staging (StagingAPIs AWS account)
```
Comment (pixeltrix) [inline README.md:114]: The staging environment in in the Mosaic-Staging account

**Review by pixeltrix — APPROVED** [review]
Just two minor amendments 👍🏻

File: README.md (hunk 2 above)
Comment (martapederiva) [reply README.md]: Done in [bd89cfe](https://github.com/LBHackney-IT/lbh-asc-brokerage-api/pull/47/commits/bd89cfe65e2b6a895e6f9b7715367ddee57b1205)

**Pull request merged**